packages/slate-editor/src/modules/editor/getEnabledExtensions.ts

```diff
@@ -146,6 +149,10 @@ export function* getEnabledExtensions({
         yield HeadingExtension();
     }
 
+    if (withInlineContacts) {
```

Comment:
I suggest adding a hard error in case both `withPressContacts` and `withInlineContacts` extensions are enabled. Just to make sure we don't shoot ourselves in foot.